python/cudf/cudf/tests/test_dataframe.py

```diff
@@ -5044,10 +5000,12 @@ def test_rowwise_ops_nullable_dtypes_partial_null(op, expected):
 
     if op in ("var", "std"):
         got = getattr(gdf, op)(axis=1, ddof=0, skipna=False)
+        expected = getattr(gdf.to_pandas(), op)(axis=1, ddof=0, skipna=False)
     else:
         got = getattr(gdf, op)(axis=1, skipna=False)
+        expected = getattr(gdf.to_pandas(), op)(axis=1, skipna=False)
 
-    assert_eq(got.null_count, expected.null_count)
+    assert_eq(got.null_count, 2)
```

Comment:
Can't we still use `expected.null_count()`? Or do we have to use a different API in 2.0 to account for NaNs vs pd.NA?

Reply:
`expected` was a `cudf.Series` previously, hence we could use `null_count`. But pandas doesn't expose any `null_count` equivalent public API/property.